samples/excel/16-custom-functions/volatile-function.yaml

```diff
@@ -7,18 +7,14 @@ api_set:
     CustomFunctionsRuntime: 1.1
 script:
     content: |
-        /** @CustomFunction */
-        function ROLL6SIDEDICE(x: number): number {
-          let i:number;
-          let result:number;
-          result = 0;
-          for (i = 0;i<x;i++) {
-            var roll:number;
-            roll = Math.random()*6;
-            roll = Math.round(roll);
-            result += roll;
-          }
-          return result;
+        /** @CustomFunction 
+        *  @description Simulates rolling a 6 sided die.
+        * @returns A whole number from 1 to 6.
+        * @volatile
+        */
+        function roll6sided(): number {
```

Comment:
The cool kids would call this `rollD6`

Reply:
I guess I'm not cool, so I wouldn't have known.  I think "roll6sided" is totally fine, to be inclusive of the uncool kids :-)

Reply:
I'd love to change this, but yeah, I'm worried many folks may think that is a cell reference. :-)